packages/components/src/Filter/FilterBar/subcomponents/FilterBarMultiSelect/FilterBarMultiSelect.tsx

```diff
@@ -94,12 +93,13 @@ export const FilterBarMultiSelect = ({
                 items
               )
             : [],
-          label: filterState.name,
+          label: label || filterState.name,
         }
 
         return filterState.isRemovable ? (
           <FilterMultiSelect.RemovableTrigger
             {...triggerProps}
+            removeButtonTooltipText={filterState.name}
```

Comment:
We don't want to be overriding the default value from this component - it should be a consumer customisation if need be.

```suggestion
```

Reply:
We need this to ensure the name of the filter is the one actually being used in the tooltip of the remove button. If a user selects `Filter 1` and we pass a label `These are your choices: `, we should, by default, have the tooltip saying `Remove filter - Filter 1` and not `Remove filter - These are your choices`.